src/analysis/inner_product_space/lax_milgram.lean

```diff
@@ -0,0 +1,142 @@
+/-
+Copyright (c) 2022 Daniel Roca González. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Roca González
+-/
+import analysis.inner_product_space.projection
+import analysis.inner_product_space.dual
+import analysis.normed_space.banach
+import topology.metric_space.antilipschitz
+
+/-!
+# The Lax-Milgram Theorem
+
+
+We consider an Hilbert space `V` over `ℝ`
+equipped with a bounded bilinear form `B : V →L[ℝ] V →L[ℝ] ℝ`.
+We define a property `is_coercive` for `B : V →L[ℝ] V →L[ℝ] ℝ`,
+such that `is_coercive B` iff
+`∃ C, (0 < C) ∧ ∀ u, C * ∥u∥ * ∥u∥ ≤ B u u`.
+
+Under the hypothesis that `B` is coercive
+we prove the Lax-Milgram theorem:
+that is, the map `continuous_linear_map_of_bilin` from `analysis.inner_product_space.dual`
+can be upgraded to a continuous equivalence `lax_milgram_equiv : V ≃L[ℝ] V`.
+
+## References
+
+* We follow the notes of Peter Howard's Spring 2020 *M612: Partial Differential Equations* lecture,
+  see[howard]
+
+## Tags
+
+dual, Lax-Milgram
+-/
+
+noncomputable theory
+
+
+
+
+open is_R_or_C continuous_linear_map linear_map linear_equiv inner_product_space
+open_locale classical complex_conjugate
+
+universe u
+
+/--
+A bounded bilinear form in an inner product space is *coercive*
+if there is some positive constant C such that `C * ∥u∥ * ∥u∥ ≤ B u u`.
+-/
+def is_coercive
+  {V : Type u} [inner_product_space ℝ V] (B : V →L[ℝ] V →L[ℝ] ℝ) : Prop :=
+∃ C, (0 < C) ∧ ∀ u, C * ∥u∥ * ∥u∥ ≤ B u u
```

Comment:
This definition would actually make sense for a general normed space over the reals.  So could you please state it in this generality and move it to `analysis.normed_space.basic`?  I don't think you need to mention it in the module docstring there, but you should adjust the module docstring here.

Reply:
In order for that to work, the operator norm is needed (to make sense of the generic equivalent of `B : V →L[ℝ] V →L[ℝ] ℝ`)). Is `analysis.normed_space.basic` really the correct location?

Reply:
You're right, sorry.  `analysis.normed_space.operator_norm`.

Reply:
Done in the last version.